scipy/stats/tests/test_qmc.py

```diff
@@ -532,9 +534,13 @@ def test_continuing(self, scramble):
         sample = engine.random(n=n_half)
         assert_allclose(sample, ref_sample[n_half:], atol=1e-1)
 
-    @pytest.mark.parametrize("scramble", scramble, ids=ids)
-    def test_reset(self, scramble):
-        engine = self.engine(d=2, scramble=scramble)
+    @pytest.mark.parametrize(
+        "scramble, seed",
+        list(zip(scramble, (170382760648021597650530316304495310428, None))),
```

Comment:
There are still a couple of cases you need to test with `scramble=True`. You need to pass `seed=None` and `seed=generator_instance`. As far as I can tell, this only tests `seed=None` with `scramble=False`.